### DISCUSSION TITLE
icmp ping

### DISCUSSION THREAD

**Discussion by amin-IT** [upptime/upptime#950] [Ideas]
hope we can have a way to ping public ip via icmp (no port)

**Comment by AnandChowdhary** [comment]
Can you do this via `node-libcurl`, `tcp-ping`, or `ws`? These are the dependencies we have available for pinging endpoints.

**Reply by amin-IT** [reply]
hi thanks for replying. I'm only trying to ping via cmd in windows which pings via icmp and doesn't use ports. anyway since my public ip can be pinged on port 53 I used it instead. Thank you
